docs/userguide/campaigns.md

```diff
@@ -1,25 +1,146 @@
-* **Conducted experiments**: A pandas ``DataFrame`` containing the experimental representation of previously conducted experiments. Preferably, the entries of this ``DataFrame`` were created by a previous campaign using [the campaigns recommend function](baybe.campaign.Campaign.recommend). If no such ``DataFrame`` is provided, it is assumed that no experiments were conducted previously.
-* **Numerical tolerance**: This is a flag for forcing numerical measurements to be within a pre-defined tolerance. Note that the setting of the tolerances is controlled as a part of the respective parameter.
-* **Previously done batches and fits** In case that a campaign builds upon previously condiucted experiments, it is possible to provide the number of previously done batches and fits.
 
-## How to use campaigns
+~~~python
+from baybe import Campaign
 
-We have a detailed example explaining how to use the ``Campaign`` object in our examples, so please see [here](./../../examples/Basics/campaign) for all necessary information.
\ No newline at end of file
+campaign = Campaign(
+    searchspace=searchspace,  # Required
+    objective=objective,  # Required
+    strategy=strategy,  # Optional
+    numerical_measurements_must_be_within_tolerance=boolean,  # Optional
+)
+~~~
+
+```{attention}
+Note that we currently also expose other fields via the constructor. This is
+only temporary, and the corresponding fields should be ignored.
+```
+
+### Creating a campaign via a JSON config
+
+It is also possible to specify a `Campaign` via a configuration string and using the
+function [`Campaign.from_config`](baybe.campaign.Campaign.from_config).
+The specification of config files as well as what they contain are automatically
+derived from the class structure.
+Furthermore, a config file can be validated using 
+[Campaign.validate_config](baybe.campaign.Campaign.validate_config).
+For more details and a full exemplary config, we refer to the corresponding
+[example](./../../examples/Serialization/create_from_config).
+
+## Getting recommendations
+
+### Basics
+
+```{attention}
+Obtaining recommendations resp. adding measurements using the `recommend` resp.
+`add_measurements` functions is the only safe way to inform a `Campaign` object about
+new measurements. These functions update the necessary metadata that is crucial for the
+proper execution of a campaign. It is important to rely on these functions to maintain
+the integrity and reliability of the campaign's execution.
+```
+
+To obtain a recommendation for the next experiment, we can query the campaign and use
+the [`recommend`](baybe.campaign.Campaign.recommend) function. The function takes only
+the `batch_quantity` keyword, specifying the desired size of
+the batch of experiments to be conducted.
+
+~~~python
+rec = campaign.recommend(batch_quantity=3)
```

Comment:
I would really like to see a table here that shows an example. That way, the user gets immediately an idea of what they will receive as output

Reply:
Disagreeing: We explicitly said that user guides should be **object-related**, not **use-case related**. If we start putting an explicit example here, we would either need to (1) adjust all of the user guide in a way that it contains a full and reasonable example, making it effectively use-case related or (2) just include a somewhat random pandas dataframe here, an option which I would not prefer. What is @Scienfitz opinion in this?

Reply:
"Basically, for each object, there is one functioning example": I thought that we do not want to have this as a requirement for the user guides? This is what the examples are for!
Also, note that this might be something specific to the campaign user guide: If we decide to have a real example here, this requires a huge amount of additional code: If we want to have an example for "This is the output of the recommendation functionality", we need to construct a full search space, specify an explicit objective and so on, blowing this up quite significantly.
And in my opinion, having tables like the ones you show here _without having them reflected in the code_ is not an option since this might confuse users even more.

Reply:
Just to make sure that we are actually talking about the same thing: You would want tables like that (properly formatted of course) in the user guide to demonstrate the output of the recommendation and adding measurements functionalities? That is how I understand your request.
![image](https://github.com/emdgroup/baybe/assets/84016733/f8f17faa-f05c-4554-8a90-663fcb31ee62)

Reply:
you dont need to create any use case and corresponding code for such table, just put a fantasy table with columns like 'MyParameter_1', 'MyParameter_2' etc or more specific ones, but the definition is not needed

Reply:
Ok, so I interpret this as "Fantasy table is fine for demonstration", and that the two of you agree on that. Then I'll create such screenshots of pandas dataframes as above and include them (since the request was specifically for the outputs of the recommend/add measurement parts, a table as the one posted above would not properly reflect the output)

Reply:
but why screenshots?

Reply:
Was about to say the same. Let's use nicely rendered tables as the one above. And I think that having more specific ones gives a better idea than just generic names. For example, the typical temperature/pressure thing we have in other places? And then there could be just a sentence above that says that indicates it's just an example...

Reply:
I think this is now already resolved, so will close